extension/src/popup/components/accountHistory/HistoryItem/index.tsx

```diff
@@ -506,33 +727,47 @@ export const HistoryItem = ({
   return (
     <div
       data-testid="history-item"
-      className="HistoryItem"
+      className="HistoryItem HistoryItem--gray12"
       onClick={() => {
         emitMetric(METRIC_NAMES.historyOpenItem);
         setDetailViewProps(txDetails);
         setIsDetailViewShowing(true);
       }}
     >
       <div className="HistoryItem__row">
-        {isLoading ? (
+        {isLoading && (
           <div className="HistoryItem__loader">
             <Loader size="2rem" />
           </div>
-        ) : (
-          <>
-            <div className="HistoryItem__icon">{renderIcon()}</div>
-            <div className="HistoryItem__operation">
-              {rowText}
-              <div className="HistoryItem__date">{dateText}</div>
+        )}
+        {!isLoading && (
+          <div className="HistoryItem__row HistoryItem--space-between">
+            <div className="HistoryItem__row">
+              <div className="HistoryItem__icon">{renderIconComponent()}</div>
+              <Text
+                as="div"
+                size="md"
+                weight="regular"
+                addlClassName="HistoryItem__description HistoryItem--gray12"
```

Comment:
similar comment as above regarding color names in class names

Reply:
https://github.com/stellar/freighter/pull/1737/commits/893112df39eb381fe715c2e68c821bdb6b58ce21